pykeeb/keyboard_matrix.py

```diff
@@ -109,3 +152,53 @@ def get_matrix(self): #needs more elegant solution, maybe using __add__?
 				if column < self.columns - 1:
 					x += self.column_hulls[row][column]
 		return x
+
+	def get_plate(self): #needs more elegant solution, maybe using __add__?
```

Comment:
I'm messing with this and get_walls() now - would these two together be the equivalent of get_matrix()?  Are you adding them just to separate the plate and case to put into separate files, or for another reason?

Reply:
yea, the tented keyboard get's a bit deep, It would be nice to have separable plate for easier wiring. I ordered a test print with a separated plate and case with the cherry mx clips and will let you know how it looks in FDM PLA from 3d Hubs.

Reply:
Okay cool.  I'm good with the functions, can you add docstrings?  I'll throw the ones you had for the arc funcs back in.